sci-visualization/mantid/mantid-3.3.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+FORTRAN_STANDARD=90
+PYTHON_COMPAT=python2_7
+inherit eutils cmake-utils fortran-2 versionator python-single-r1
+
+MAJOR_PV=$(get_version_component_range 1-2)
+
+DESCRIPTION="Utilities for processing and plotting neutron scattering data"
+HOMEPAGE="http://www.mantidproject.org/"
+SRC_URI="mirror://sourceforge/project/${PN}/${MAJOR_PV}/${P}-Source.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc opencl paraview shared-libs tcmalloc test"
+RESTRICT="test" # Testing requires sample data and X11 access
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=sci-libs/nexus-4.2
+	>=dev-libs/poco-1.4.2
+	dev-libs/boost[python]
+	opencl?		( virtual/opencl )
+	tcmalloc?	( dev-util/google-perftools )
+	paraview?	( >=sci-visualization/paraview-4 )
+	virtual/opengl
+	dev-qt/qthelp
+	x11-libs/qscintilla
+	x11-libs/qwt:5
+	x11-libs/qwtplot3d
+	dev-python/PyQt4
+	sci-libs/gsl
+	dev-python/ipython[qt4]
+	dev-python/numpy
+	sci-libs/scipy
+	dev-cpp/muParser
+	dev-libs/jsoncpp
+	dev-libs/openssl
+	sci-libs/opencascade
+"
+
+DEPEND="${RDEPEND}
+	dev-python/sphinx
+	doc?	( app-doc/doxygen
+		  dev-python/sphinx
+		  dev-python/sphinx-bootstrap-theme )
+	test?	( dev-util/cppcheck )
+"
```

Comment:
I don't have much experience with the python-single-r1.eclass, but I think the use of [PYTHON_USEDEP](http://wiki.gentoo.org/wiki/Project:Python/python-single-r1#PYTHON_USEDEP) mandatory.